backend/src/v5/routes/users.js

```diff
@@ -103,11 +165,176 @@ const establishRoutes = () => {
+	 *
+	 */
+	router.get('/user', validSession, getProfile);
+
+	/**
+	* @openapi
+	* /user:
+	*   put:
+	*     description: Updates the profile of the logged in user
+	*     tags: [Auth]
+	*     operationId: updateProfile
+	*     requestBody:
+	*       content:
+	*         application/json:
+	*           schema:
+	*             type: object
+	*             properties:
+	*               firstName:
+	*                 type: string
+	*                 description: The first name of the user
+	*                 example: Jason
+	*               lastName:
+	*                 type: string
+	*                 description: The last name of the user
+	*                 example: Voorhees
+	*               email:
+	*                 type: string
+	*                 description: The email of the user
+	*                 example: jason@vorhees.com
+	*                 format: email
+	*               oldPassport:
+	*                 type: string
+	*                 description: The old password of the user
+	*                 example: password12345
+    *                 format: password
+	*               newPassport:
+	*                 type: string
+	*                 description: The new password of the user
+	*                 example: password12345
+    *                 format: password
+	*     responses:
+	*       401:
+	*         $ref: "#/components/responses/notLoggedIn"
+	*       200:
+	*         description: Updates the details of the user
+	*
+	*/
+	router.put('/user', isLoggedIn, validateUpdateData, updateProfile);
+
+	/**
+	* @openapi
+	* /user/key:
+	*   post:
+	*     description: Generates a new API key for the logged in user
+	*     tags: [Auth]
+	*     operationId: generateApiKey
+	*     responses:
+	*       401:
+	*         $ref: "#/components/responses/notLoggedIn"
+	*       200:
+	*         description: Updates the details of the user
+	*         content:
+	*           application/json:
+	*             schema:
+	*               type: object
+	*               properties:
+	*                 apiKey:
+	*                   type: string
+	*                   description: The new API key of the user
+	*                   example: 20f947a673dce5419ce187ca7998a68f
+	*/
+	router.post('/user/key', isLoggedIn, generateApiKey);
+
+	/**
+	* @openapi
+	* /user/key:
+	*   delete:
+	*     description: Deletes the API key of the logged in user
+	*     tags: [Auth]
+	*     operationId: deleteApiKey
+	*     responses:
+	*       401:
+	*         $ref: "#/components/responses/notLoggedIn"
+	*       200:
+	*         description: Deletes the details of the user
+	*/
+	router.delete('/user/key', isLoggedIn, deleteApiKey);
+
+	/**
+	* @openapi
+	* /user/avatar:
+	*   get:
+	*     description: Gets the avatar of the logged in user
+	*     tags: [Auth]
+	*     operationId: getAvatar
+	*     responses:
+	*       401:
+	*         $ref: "#/components/responses/notLoggedIn"
+	*       200:
+	*         description: Updates the details of the user
```

Comment:
this doesn't render:
![image](https://user-images.githubusercontent.com/11945337/149936080-f9e69d0a-6fcb-45c3-8ebe-42383c6d3083.png)